sorl/thumbnail/engines/base.py

```diff
@@ -27,6 +28,10 @@ def create(self, image, geometry, options):
         image = self.padding(image, geometry, options)
         return image
 
+    def opaquen(self, image, options):
+        """Wrapper for ``_opaquen``."""
+        return self._opaquen(image, options)
```

Comment:
This should do a test whether or not `_opaquen` should be called.  I believe making the image opaque is only needed when converting an image with transparency to a format that doesn't have it, like PNG to JPEG.  However, I don't think it's needed if it's PNG to PNG; I believe transparency gets correctly preserved in that case scenario.